packages/create-wagmi/src/index.ts

```diff
@@ -0,0 +1,246 @@
+        )
+    }
+
+    ////////////////////////////////////////////////////////////////
+
+    log()
+    log(chalk.magenta('♥ gm, welcome to wagmi ♥'))
+    log()
+
+    ////////////////////////////////////////////////////////////////
+
+    let projectPath
+    let projectName
+    if (args[0]) {
+      projectPath = args[0]
+      const splitPath = args[0].split('/')
+      projectName = splitPath[splitPath.length - 1]
+    }
+
+    if (!projectName) {
+      projectName = (
+        await prompts({
+          initial: 'my-wagmi-app',
+          name: 'projectName',
+          message: 'What would you like to name your project?',
+          type: 'text',
+          validate: (name) =>
+            !validatePackageName(name).valid
+              ? `"${name}" is not a valid project name. Enter another name.`
+              : true,
+        })
+      ).projectName
+      if (!projectName) throw new CLIError()
+      projectPath = projectName
+      log(chalk.cyan('👍 Sick name'))
+      log()
+    }
+
+    if (!validatePackageName(projectName).valid)
+      throw new CLIError(
+        [
+          chalk.red(`🙈 "${projectName}" is not a valid project name.`),
+          validatePackageName(projectName).warnings?.map(
+            (warning) => `👉 ${warning}`,
+          ),
+        ].join('\n'),
+      )
+
+    ////////////////////////////////////////////////////////////////
+
+    const targetPath = path.join(process.cwd(), projectPath)
+
+    if (existsSync(targetPath))
+      throw new CLIError(
+        [
+          chalk.red(`🙈 the directory "${projectPath}" already exists.`),
+          `👉 choose another name or delete the directory.`,
+        ].join('\n'),
+      )
+
+    ////////////////////////////////////////////////////////////////
+
+    if (!templateName) {
+      templateName = (
+        await prompts({
+          name: 'templateName',
+          message: 'What template would you like to use?',
+          type: 'select',
+          choices: templates.map(({ description, name, title }) => ({
+            description,
+            title,
+            value: name,
+          })),
+        })
+      ).templateName
+    }
+
+    ////////////////////////////////////////////////////////////////
+
+    log(chalk.cyan('👷‍♂️ Creating a new wagmi app in', chalk.green(targetPath)))
+    log()
+
+    const templatePath = path.join(templatesPath, templateName)
+    await cpy(path.join(templatePath, '**', '*'), targetPath, {
+      rename: (name) => name.replace(/^_dot_/, '.'),
+    })
+
+    const packageJson = await readJSON(path.join(targetPath, 'package.json'))
+    packageJson.name = projectName
+    await writeFile(
+      path.join(targetPath, 'package.json'),
+      JSON.stringify(packageJson, null, 2),
+    )
+
+    ////////////////////////////////////////////////////////////////
+
+    const packageManager = options.pnpm
+      ? 'pnpm'
+      : options.yarn
+      ? 'yarn'
+      : options.npm
+      ? 'npm'
+      : await detect()
```

Comment:
This `detect` I don't think worked for me (used `pnpm create wagmi`, but installed with `yarn`). [This package](https://github.com/antfu/ni) could be an alternative.